tests/python/relay/test_backend_graph_executor.py

```diff
@@ -148,6 +148,12 @@ def test_plan_memory():
     assert len(device_types) == 1
     assert len(storage_sizes) == 4
 
+    # Check the specific size of each sid
```

Comment:
might be better to assert on storage_sizes all at once, would provide a better failure message in CI